app/src/main/java/de/danoeh/antennapod/view/ChapterSeekBar.java

```diff
@@ -75,8 +81,8 @@ public void run() {
 
     @Override
     protected synchronized void onDraw(Canvas canvas) {
-        top = getTop() + density * 7.5f;
-        bottom = getBottom() - density * 7.5f;
+        top = getTop() + density * 7.5f * versionScale;
+        bottom = getBottom() - density * 7.5f * versionScale;
```

Comment:
If I read this correctly, you take the top and bottom of the view and go inwards in order to calculate the size of the bar. I fear that some vendors with their custom themes might break this, too, not only on API<23. Wouldn't it be more reliable to calculate the center of the View and then go outwards from there? That way, the bar always has exactly the same height, independently of the height of the View.

Reply:
Thanks :)